unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Main/Main.cs

```diff
@@ -96,13 +93,19 @@ private void Update()
             featureController?.Update();
         }
 
-        private void LateUpdate() { Environment.i.world.sceneController.LateUpdate(); }
+        private void LateUpdate()
+        {
+            Environment.i.world.sceneController.LateUpdate();
+            featureController?.LateUpdate();
```

Comment:
`featureController` will never be null. We really want to check for nullity every frame here?